indra/newview/llstartup.cpp

```diff
@@ -2451,6 +2448,23 @@ bool idle_startup()
 
         LLPerfStats::StatsRecorder::setAutotuneInit();
 
+        // Display Avatar Welcome Pack the first time a user logs in
+        // (or clears their settings....)
+        if (gSavedSettings.getBOOL("FirstLoginThisInstall"))
```

Comment:
My biggest worry is gonna be this getting annoying to existing users - but otherwise seems fine.